src/pkg/ocidriver/ocimanifest.go

```diff
@@ -0,0 +1,33 @@
+package ocidriver
```

Comment:
Rename the package name please.
The `OCI` is the name of an organization not the registry